tls/s2n_server_cert.c

```diff
@@ -34,13 +31,14 @@ int s2n_server_cert_recv(struct s2n_connection *conn)
     uint32_t size_of_all_certificates;
     POSIX_GUARD(s2n_stuffer_read_uint24(&conn->handshake.io, &size_of_all_certificates));
 
-    S2N_ERROR_IF(size_of_all_certificates > s2n_stuffer_data_available(&conn->handshake.io) || size_of_all_certificates < 3, S2N_ERR_BAD_MESSAGE);
+    S2N_ERROR_IF(size_of_all_certificates > s2n_stuffer_data_available(&conn->handshake.io) || size_of_all_certificates < 3,
+            S2N_ERR_BAD_MESSAGE);
```

Comment:
formatting looks odd. wondering if its one of the changes you plan to address later.